### PR TITLE
LPS - 151330 & LPS - 151630 [CP 2.0] - Deactivate key before selecting any keys - DXP 

### PR DESCRIPTION
**Pull request by claricebrasil**
When a user select to deactivate in the DXP activation table menu dropdown, it should be redirected to another route and select a key to deactivate.

### CODE REVIEW

**Review by thaynam — APPROVED** [review]

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
Hi @claricebrasil! Congrats for you job! I suggest just a simple change about React Fragment and when we can use it :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DeactivateKeysTable/components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+					/>
+				</div>
+
+				<BadgeFilter
+					activationKeysLength={activationKeys?.length}
+					filtersState={[filters, setFilters]}
+					loading={loading}
+				/>
+			</div>
+		</>
```
[the remark this answers is not in the document]
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DeactivateKeysTable/components/Header/index.js:38]: Hey, Jair!! You're right, already fixed it! Thank you!

**Review by igordmf — APPROVED** [review]
Good job!

**Review by joaosvassis — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**